.gitmodules

```diff
@@ -62,10 +62,6 @@
     ignore = dirty
     path = third_party/ideep
     url = https://github.com/intel/ideep
-[submodule "third_party/nccl/nccl"]
-    ignore = dirty
-    path = third_party/nccl/nccl
-    url = https://github.com/NVIDIA/nccl
```

Comment:
hmm, is this a complete removal? (as replaced by `checkout_nccl()` below?)

Reply:
Yes complete removal, we replace by checkout_nccl()